content-node/docker-compose/README.md

```diff
@@ -2,71 +2,58 @@
 
 This repository contains the necessary files to deploy a Content Node using Docker Compose.
 
-## Prerequisites
+This README assumes you have followed the [Content Node Setup Guide](https://docs.earthfast.com/node-operators/content-node-setup) and satisfied all prerequisites & registered your Content Node.
 
-- A Linux-based system (preferably Ubuntu)
-- An internet-connected server with a static public IP address
-- A DNS record pointing to your server's IP address eg `content1.us-central1-a.testnet-sepolia.earthfastnodes.com`
-- A Registered Content Node --> https://docs.earthfast.com/node-operators/content-node-setup#register-your-nodes
+## Setup
 
-## Quick Start
-
-#### 1. Clone this repository and run the setup script:
+Clone this repository and run the setup script:
 ```sh
-git clone https://github.com/earthfast/earthfast-node-setup-examples.git
-cd earthfast-node-setup-examples/content-node/docker-compose
-./setup.sh
-```
-
-You can use the `--staging` flag for sepolia staging instead.
+> git clone https://github.com/earthfast/node-operator-tooling
+> cd node-operator-tooling/content-node/docker-compose
+> ./setup.sh
 ```
-./setup.sh --staging
-```
-
-##### SSL Configuration
 
-If `SETUP_SSL` is set to 'true', the setup will automatically configure SSL using Let's Encrypt. Make sure the url from `SERVER_NAME` correctly pointed to your server's IP address before enabling this option.
+The `setup.sh` script will prompt you for environment variables:
+* `SERVER_NAME` = your node's FQDN
+* `NODE_ID` = the NodeID outputted in Step 5 above
+* `SETUP_SSL` = if you have configured this externally, set this to false. Setting it to true will locally configure SSL with Let's Encrypt. Make sure the url from SERVER_NAME is correctly pointed to your server's IP address before enabling this option.
```

Comment:
"if you have configured HTTPS/SSL externally, set this to false."

Reply:
if a user selects SETUP_SSL=true, we should check in setup.py if the FQDN points to this server. not a blocker to merge this PR but probably good developer experience

Reply:
updated readme, added [linear card](https://linear.app/earthfast/issue/ENG-234/if-setup-ssl=true-check-in-setuppy-if-fqdn-points-to-vm-ip)